cmd/global-query/cmd/root.go

```diff
@@ -126,7 +126,11 @@ func initQuerier() (distributed.Querier, error) {
 	querierType := viper.GetString(conf.QuerierType)
 	switch querierType {
 	case string(distributed.APIClientQuerierType):
-		return distributed.NewAPIClientQuerier(viper.GetString(conf.QuerierConfig))
+		return distributed.NewAPIClientQuerier(
```

Comment:
Oo - Probably need to split the `New...()` and `SetMaxConcurrent()` part to honor the return pattern...

Reply:
Doh!